### PR TITLE
Kinodynamic

### PR DESCRIPTION
**Pull request by pFernbach**
Add a tutorial script showing the new functionality introduced by the PR https://github.com/humanoid-path-planner/hpp-core/pull/117

It also show the use of the methods to display paths and roadmap (not restricted to kinodynamic methods)

### CODE REVIEW

File: script/tutorial_kinodynamic.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+aMax=1.
+vMax=2.
+robot.client.robot.setExtraConfigSpaceBounds([-vMax,vMax,-vMax,vMax,0,0,-aMax,aMax,-aMax,aMax,0,0])
+
+from hpp.corbaserver import ProblemSolver
+ps = ProblemSolver (robot)
+# define the velocity and acceleration bounds used by the steering method. This bounds will be stastified along the whole trajectory.
+ps.client.problem.setParameter("Kinodynamic/velocityBound",omniORB.any.to_any(vMax))
+ps.client.problem.setParameter("Kinodynamic/accelerationBound",omniORB.any.to_any(aMax))
+ps.client.problem.setParameter("PathOptimization/RandomShortcut/NumberOfLoops",omniORB.any.to_any(100))
```
[the remark this answers is not in the document]
Comment (pFernbach) [reply script/tutorial_kinodynamic.py]: Done in https://github.com/humanoid-path-planner/hpp_tutorial/pull/14/commits/1b6b5b2aa3b5663685e7a038492f1b0e7dbfa11a

File: script/tutorial_kinodynamic.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  def __init__ (self, robotName, load = True):
+        Robot.__init__ (self, robotName, self.rootJointType, load)
+        self.tf_root = "base_footprint"
+
+robot = RobotRod("rod")
+
+robot.setJointBounds ("root_joint", [-7, 6.5, -7, 7,0.4,0.4])
+
+# Kinodynamic methods need at least 6 extraConfigs, to store the velocity (first 3) and acceleration (last 3) of the root
+robot.client.robot.setDimensionExtraConfigSpace(6)
```
[the remark this answers is not in the document]
Comment (pFernbach) [reply script/tutorial_kinodynamic.py]: No, the current implementation can only deal with the translation part of the freeflyer. The value 6 come from 3 values for the 3D velocity and 3 for the acceleration. I added a header in the tutorial explaining this.

File: script/tutorial_kinodynamic.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jmirabel) [reply script/tutorial_kinodynamic.py]: :+1:

File: script/tutorial_kinodynamic.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jmirabel) [reply script/tutorial_kinodynamic.py]: Understood.

**Pull request merged**